snuba/clickhouse/native.py

```diff
@@ -23,6 +26,13 @@
 metrics = MetricsWrapper(environment.metrics, "clickhouse.native")
 
 
+@dataclass(frozen=True)
+class ClickhouseResult:
+    results: Sequence[Any] = field(default_factory=list)
+    meta: Sequence[Any] | None = None
+    profile: Mapping[str, Any] | None = None
```

Comment:
Can we provide something more specific ?
Like having a datclass for the profile information. I am not sure what we gain from leaving it as a free form mapping knowing that the profile information is static and well known.

Reply:
I removed the profiling data from this PR, it's not really this is about. I'll add it back in a separate PR.